2.ensemble-z-analysis/scripts/train_models_given_z.py

```diff
@@ -55,6 +59,65 @@
 out_dir = args.out_dir
 num_seeds = int(args.num_seeds)
 shuffle = args.shuffle
+subset_mad_genes = int(args.subset_mad_genes)
+
+
+def get_recon_correlation(df, recon_mat_dict, algorithm, cor_type, axis):
+    """
+    Get gene or sample correlations between input and reconstructed input
+
+    Arguments:
+    df - the input dataframe
+    recon_mat_dict - dictionary of different algorithms reconstructions
+    algorithm - string representing the compression algorithm
+    cor_type - string representing Pearson or Spearman correlations
+    axis - string representing genes or samples to calculate correlations for
```

Comment:
Is there going to be future options for the axis parameter? if not wouldn't it be easier to just make a boolean?

Reply:
yeah, that is a good point. I will update!